src/Contract/RestInterface.php

```diff
@@ -23,4 +24,9 @@ public function postJson(string $uri, string $jsonBody, array $headers = []): Re
      * @param array<string, mixed> $headers
      */
     public function postForm(string $uri, array $formFields, array $headers = []): ResponseInterface;
+
+    /**
+     * @return array<string, string>
+     */
+    public function getAuthorizationHeadersFromToken(Token $token): array;
```

Comment:
It was a difficult decision, but I finally moved this code here. Previously it was part of the `TokenAwareTrait`.